brig/cmd/brig/commands/build_get.go

```diff
@@ -47,6 +49,18 @@ func getBuild(out io.Writer, bid string) error {
 	script := string(b.Script)
 	payload := string(b.Payload)
 
+	if output == "json" {
+		b, err := json.MarshalIndent(b, "", "    ")
+		if err != nil {
+			return err
+		}
+		_, err = out.Write(b)
+		return err
+	}
+
+	// TODO
+	//
+	// why are we setting these to nil?
```

Comment:
Here we set the script and payload displayed on the console to be nil.
Any idea why we do that for `build get` and not other commands?

Reply:
Once we figure this out, I can either modify the code or remove / improve the comment.

Reply:
Yea, it looks like they are removed for default output but then if the global verbose flag is set, it adds them back in: https://github.com/brigadecore/brigade/blob/master/brig/cmd/brig/commands/build_get.go#L58-L60

Perhaps we can add/improve the comment here?

Reply:
Oh, the it was just below..
Ok, I added a comment there.